airflow/providers/amazon/aws/operators/sqs.py

```diff
@@ -89,3 +90,19 @@ def execute(self, context: 'Context'):
         self.log.info('result is send_message is %s', result)
 
         return result
+
+
+class SQSPublishOperator(SqsPublishOperator):
+    """
+    This operator is deprecated.
+    Please use :class:`airflow.providers.amazon.aws.operators.sqs.SqsPublishOperator`.
+    """
+
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
+            "This operator is deprecated. "
+            "Please use :class:`airflow.providers.amazon.aws.operators.sqs.SqsPublishOperator`.",
```

Comment:
```suggestion
            "Please use `airflow.providers.amazon.aws.operators.sqs.SqsPublishOperator`.",
```

Deprecation warnings are not rst-rendered